config/locales/diaspora/en.yml

```diff
@@ -1091,18 +1091,23 @@ en:
 
   registrations:
     new:
+      title: "Join the community!"
+      subtitle: "You're about to create an account on %{podname}. One diaspora* account is enough to reach the whole network, you don't need to create a new account here if you already have one on another pod."
       enter_email: "Enter your email address"
-      enter_username: "Pick a username (only letters, numbers, and underscores)"
+      enter_username: "Pick a username (only letters, numbers, _ and -)"
       enter_password: "Enter a password (six character minimum)"
       enter_password_again: "Enter the same password as before"
       sign_up: "Create account"
       email: "Email"
+      email_advice: "Your e-mail will never be shared with anyone and will only be used to allow you to recover your account or send notifications."
```

Comment:
I suggest reordering as follows: "Your e-mail will never be shared with anyone and will only be used to send notifications and to enable you to recover your account."

Reply:
> notifications could be seen as spam

Notifications are entirely optional, so not spam. If you are concerned about that, you could expand this to 'to send the notifications you request'. I think notifications should go first as that's the normal/most common use for the email address, whereas recovering the account is a (hopefully) rare thing. At the very least, please change 'or send notifications' at the end to 'and to send notifications'.

Reply:
Yeah I changed it to "Your e-mail will never be shared with anyone and will only be used to enable you to recover your account and to send notifications."